### PR TITLE
Add JSX elements conveying HTML semantic meaning

### PR DESCRIPTION
**Pull request by tbguest**
I'm working with state hooks now, but thought it best to compartmentalise by pushing one task at a time.

### CODE REVIEW

File: tristan/ding-wall/src/components/AddFooter.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './AddFooter.css';
+
+function AddFooter() {
+  return (
+    <div className="footer">
```
Comment (lauchness) [inline tristan/ding-wall/src/components/AddFooter.js]: No need to wrap the footer in a div, just apply the className to footer directly

- [ ] Remove wrapping div

File: tristan/ding-wall/src/components/AddHeader.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './AddHeader.css';
+
+function AddHeader() {
+  return (
+    <div className="header-container">
```
Comment (lauchness) [inline tristan/ding-wall/src/components/AddHeader.js]: Same comment as footer.js

- [ ] Remove wrapping div

File: tristan/ding-wall/src/components/AsideBlurb.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+function AsideBlurb() {
+  return (
+    <div className="aside-container">
+      <aside>
+        <h3>The house:</h3>
+        <p>Purchased in fall 2019, our house is in the midst of a major renovation.</p>
+        <p>The house is old as hell, and the walls contain a million little rodent poops. Non-dingwall-type people would have opted to tear the place down, but we decided to be all "romantic" about it, and retain what solid wood remained.</p>
+      </aside>
+    </div>
```
Comment (lauchness) [inline tristan/ding-wall/src/components/AsideBlurb.js]: Same comment as footer.js. I'll add a suggested change here to show what I mean
```suggestion
      <aside className="aside-container">
        <h3>The house:</h3>
        <p>Purchased in fall 2019, our house is in the midst of a major renovation.</p>
        <p>The house is old as hell, and the walls contain a million little rodent poops. Non-dingwall-type people would have opted to tear the place down, but we decided to be all "romantic" about it, and retain what solid wood remained.</p>
      </aside>
```

File: tristan/ding-wall/src/components/ListSection.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './ListSection.css';
+
+function ListSection() {
+  return (
+    <div className="list-container">
```
Comment (lauchness) [inline tristan/ding-wall/src/components/ListSection.js]: Same comment as footer.js

- [ ] Remove wrapping div

File: tristan/ding-wall/src/components/MainContent.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './MainContent.css';
+import ListSection from './ListSection';
+import AsideBlurb from './AsideBlurb';
+
+function MainContent() {
+  return (
+    <div className="main-container">
```
Comment (lauchness) [inline tristan/ding-wall/src/components/MainContent.js]: Same as footer.js

- [ ] Remove wrapping div

**Comment by tbguest** [comment]
I committed the changes. It looks like they show up automatically in this pull request so I'll assume no further action is required unless directed by you.

**Pull request merged**